### PR TITLE
Issue/tdd piece placement

### PR DESCRIPTION
**Pull request by owais34**
Added PiecePlacement class according to the specifications , added unit test file for the same with basic tests. Need suggestions to add more test scenarios.

### CODE REVIEW

File: src/FEN/field/PiecePlacement.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import UnknownNotationError from "../../error/UnknownNotationError";
+import ValidationInterface from "../ValidationInterface";
+
```
Comment (programarivm) [inline src/FEN/field/PiecePlacement.ts:3]: There should be one line break between the last import statement and the class declaration.

File: src/FEN/field/PiecePlacement.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+class PiecePlacement implements ValidationInterface{
+  validate = (value: string): string => {
+    const fields=value.split("/");
+    const count=fields.length;
+    if(count===8){
```
Comment (programarivm) [inline src/FEN/field/PiecePlacement.ts:9]: Blank spaces should be used when writing control flow statements and blocks as per the style guide. See https://google.github.io/styleguide/tsguide.html#control-flow-statements-blocks

File: src/FEN/field/PiecePlacement.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import UnknownNotationError from "../../error/UnknownNotationError";
+import ValidationInterface from "../ValidationInterface";
+
+
+class PiecePlacement implements ValidationInterface{
+  validate = (value: string): string => {
+    const fields=value.split("/");
+    const count=fields.length;
```
Comment (programarivm) [inline src/FEN/field/PiecePlacement.ts:8]: Blank spaces should be used when assigning values to variables for better readability as in this example. See https://google.github.io/styleguide/tsguide.html#control-flow-statements-blocks

**Comment by programarivm** [comment]
Thanks for sending this PR.

Looks good! Please let me just point out the following for future PRs:

- The name of the PR branch should be `issue/7-tdd-piece-placement`
- On this occasion, let's remove `PiecePlacementTest.ts` for the time being because there's no `PiecePlacementTest.php` file on the PHP Chess repo to be transcribed into TypeScript. Sorry that I just didn't realize this when opening the issue. 

Keep it up and happy learning and coding.

**Comment by owais34** [comment]
Will keep that in mind . Thanks.

**Pull request closed without merging**